app/markets/ActivityTable.tsx

```diff
@@ -38,7 +39,7 @@ const ITEMS_PER_PAGE = 10;
 export const ActivityTable = ({ id }: { id: string }) => {
   const [page, setPage] = useState(1);
 
-  const { data, isLoading } = useQuery({
+  const { data: txs, isLoading: isTxsLoading } = useQuery({
     queryKey: ['getMarketTransactions', id, page - 1],
```

Comment:
Let's delete this query and just make a query to `getMarketTrades`. Let's only iterate through  `trades`

Reply:
Removed!